src/View/InputView.js

```diff
@@ -0,0 +1,14 @@
+const { MESSAGE } = require('../libs/const');
+const { Console } = require('@woowacourse/mission-utils');
+
+class InputView {
+  static readPlayerAnswer(callback) {
+    Console.readLine(MESSAGE.inputNumber, callback);
+  }
+
+  static readCommand(callback) {
+    Console.readLine(MESSAGE.inputOption, callback);
+  }
+}
```

Comment:
InputView는 Class로 선언됐지만 데이터를 다루지 않는 클래스, 다시 말해 필드가 없이 메소드만 제공하는 클래스이므로 여기에 정적 메소드를 활용하는 것보다는 객체로서 활용하는 것이 더 좋아보여요👍

```suggestion
const InputView = {
  readPlayerAnswer(callback) {
    Console.readLine(MESSAGE.inputNumber, callback);
  }

  readCommand(callback) {
    Console.readLine(MESSAGE.inputOption, callback);
  }
}
```

Static Method를 활용할 때에는 특정 클래스 인스턴스가 아니라 모든 클래스 인스턴스에 필요할 때 유용하게 쓰일 수 있습니다!

Reply:
네! 맞습니다ㅎㅎ `InputView`, `OutputView`가 굳이 클래스여야 하나? 라는 의문이 들더라고요! 객체로 활용해야 겠습니다ㅎㅎ